File: SDKs/tfl.gov.uk/v1/python/src/sdk/models/shared/tfl_api_presentation_entities_routesectionnaptanentrysequence.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class TflAPIPresentationEntitiesRouteSectionNaptanEntrySequence:
    ordinal: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('ordinal') }})
    stop_point: Optional[TflAPIPresentationEntitiesStopPoint] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('stopPoint') }})
    

File: SDKs/tfl.gov.uk/v1/typescript/src/sdk/models/shared/tflapipresentationentitiesroutesectionnaptanentrysequence.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { TflApiPresentationEntitiesStopPoint } from "./tflapipresentationentitiesstoppoint";



export class TflApiPresentationEntitiesRouteSectionNaptanEntrySequence extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=ordinal" })
  ordinal?: number;

  @SpeakeasyMetadata({ data: "json, name=stopPoint" })
  stopPoint?: TflApiPresentationEntitiesStopPoint;
}
